docs/src/SUMMARY.md

```diff
@@ -1,38 +1,42 @@
 
-- [Introduction](./introduction.md)
-- [Quickstart](./quickstart.md)
-    - [Netcat](./quickstarts/netcat.md)
-    - [Agones + Xonotic (Sidecar)](./quickstarts/agones-xonotic-sidecar.md)
-    - [Agones + Xonotic (xDS)](./quickstarts/agones-xonotic-xds.md)
-- [Usage](./using.md)
-  - [File Configuration](./file-configuration.md)
-- [Integrations](./integrations.md)
-- [Proxy Mode](./proxy-mode.md)
-  - [Concepts](proxy/concepts.md)
-  - [Filters](./filters.md)
-      - [Capture](proxy/filters/capture.md)
-      - [Compress](proxy/filters/compress.md)
-      - [Concatenate Bytes](proxy/filters/concatenate_bytes.md)
-      - [Debug](proxy/filters/debug.md)
-      - [Drop](proxy/filters/drop.md)
-      - [Firewall](proxy/filters/firewall.md)
-      - [Load Balancer](proxy/filters/load_balancer.md)
-      - [Local Rate Limit](proxy/filters/local_rate_limit.md)
-      - [Match](proxy/filters/match.md)
-      - [Pass](proxy/filters/pass.md)
-      - [Timestamp](proxy/filters/timestamp.md)
-      - [Token Router](proxy/filters/token_router.md)
-      - [Writing Custom Filters](proxy/filters/writing_custom_filters.md)
-  - [Metrics](proxy/metrics.md)
-  - [Maxmind](proxy/maxmind.md)
-- [xDS Management API](./xds.md)
-  - [Built-in xDS Providers](./xds/builtin.md)
-    - [Filesytem](./xds/providers/filesystem.md)
-    - [Agones](./xds/providers/agones.md)
-  - [Metrics](./xds/metrics.md)
-- [Administration](./admin.md)
-- [Client SDKs](./sdks.md)
-    - [Unreal Engine](./sdks/unreal-engine.md)
-- [Examples](./examples.md)
-- [FAQ](./faq.md)
+# Services
+- [Proxy](./services/proxy.md)
+    - [Filters](./services/proxy/filters.md)
+        - [Capture](./services/proxy/filters/capture.md)
+        - [Compress](./services/proxy/filters/compress.md)
+        - [Concatenate Bytes](./services/proxy/filters/concatenate_bytes.md)
+        - [Debug](./services/proxy/filters/debug.md)
+        - [Drop](./services/proxy/filters/drop.md)
+        - [Firewall](./services/proxy/filters/firewall.md)
+        - [Load Balancer](./services/proxy/filters/load_balancer.md)
+        - [Local Rate Limit](./services/proxy/filters/local_rate_limit.md)
+        - [Match](./services/proxy/filters/match.md)
+        - [Pass](./services/proxy/filters/pass.md)
+        - [Timestamp](./services/proxy/filters/timestamp.md)
+        - [Token Router](./services/proxy/filters/token_router.md)
+        - [Writing Custom Filters](./services/proxy/filters/writing_custom_filters.md)
+    - [Metrics](./services/proxy/metrics.md)
+
+---
+
+- [Control Plane](./services/xds.md)
+    - [Metrics](./services/xds/metrics.md)
+    - [Providers]()
+        - [Agones](./services/xds/providers/agones.md)
+        - [Filesystem](./services/xds/providers/filesystem.md)
+
+# SDKs
+- [Unreal Engine](./sdks/unreal-engine.md)
+
+# Deployment
+- [Administration](./deployment/admin.md)
+- [Architecture Examples](./deployment/examples.md)
+- [Configuration](./deployment/configuration.md)
+
+# Quickstart Guides
```

Comment:
I'd be tempted to to move the quickstarts to before **Services**.

If we look at Google Cloud doc's as a reference, Quickstarts always come towards the top, as they are the easy path to getting started with the product.

For example:
https://cloud.google.com/kubernetes-engine/docs/deploy-app-cluster

![image](https://user-images.githubusercontent.com/298370/214449152-cbe4e1cd-6236-429c-9126-09ac32489455.png)